### ISSUE TITLE
Randomization Delays

### ISSUE THREAD

**Issue by slunari** [jacobjude/Toontown-Rewritten-Bot#1]
Is it intended for your settings to be set to WASD before running this feature? If not, it seems to just be spamming "ad" and "da"

**Comment by jjacob-0** [comment]
Yeah, the movement keys should be set to WASD instead of the arrow keys. I'll add an option that allows you to use arrow keys soon.

**Comment by slunari** [comment]
Oh cool, thanks! Any possible plans for golfing automation / racing automation in the future? I haven't seen anything like that done before.

**Comment by jjacob-0** [comment]
Probably one day, but I don't have any current plans.
